kura/test/org.eclipse.kura.container.orchestration.provider.test/src/test/java/org/eclipse/kura/container/orchestration/provider/ContainerOrchestrationServiceImplTest.java

```diff
@@ -663,6 +664,11 @@ private void whenStopContainer() throws KuraException {
         this.dockerService.stopContainer(this.containerId);
     }
 
+    private void whenDeleteContainer() throws KuraException {
+        // deleteContainer
+        this.dockerService.deleteContainer(this.containerId);
```

Comment:
Is this comment really needed though? 😄 

```suggestion
        this.dockerService.deleteContainer(this.containerId);
```